src/ext/coms.c

```diff
@@ -497,16 +497,19 @@ void ddtrace_msgpack_group_stack_by_id(ddtrace_coms_stack_t *stack, struct _grou
 void *ddtrace_init_read_userdata(ddtrace_coms_stack_t *stack) {
     size_t total_bytes = atomic_load(&stack->bytes_written);
 
-    struct _grouped_stack_t *readstack_ptr = malloc(sizeof(struct _grouped_stack_t));
-    struct _grouped_stack_t readstack = {.position = 0, .total_bytes = total_bytes};
+    struct _grouped_stack_t *readstack = calloc(sizeof(struct _grouped_stack_t), 1);
```

Comment:
Any particular reason this switched from `malloc(x) to `calloc(x, 1)`?

Reply:
to initialize all values to 0, before it was set somewhat manually.